packages/landing/src/components/map.tsx

```diff
@@ -204,6 +213,13 @@ export class Basemaps extends Component<unknown, { isLayerSwitcherEnabled: boole
         Config.map.on('visibleLayers', this.updateVisibleLayers),
       );
 
+      const terrain = this.map.getStyle().terrain;
+      console.log(terrain);
+      if (terrain != null) {
```

Comment:
this should be similar to `ensureGeoControl` where it adds and removes based on the current layer.